include/odp/api/spec/init.h

```diff
@@ -341,6 +342,28 @@ int odp_term_local(void);
  */
 int odp_term_global(odp_instance_t instance);
 
+/**
+ * Abnormally terminate ODP application after a non-recoverable error
+ *
+ * Use to notify a fatal non-recoverable error and finally terminate execution. Calling this
```

Comment:
I kind of like the "notify" here as the resulting behavior might be a bunch teardown procedures, i.e. application notifies the condition, then stuff happens and then eventually process terminates.

Reply:
Also to me "notify" seems too light weight term, since from ODP implementation point of view the calling thread terminates due to the call (abort will be called).

May be something like: "This function may be used to abnormally terminate the calling thread and finally the entire application.  Application is not expected to carry out normal local/global clean-up after one of its threads have done a panic call."

Reply:
Well, the "notify" doesn't have to "carry the weight", thread-stopping and others are separately documented. Anyways, there are now two complaints so I'll change the text a bit then.